var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -105,7 +122,8 @@ class Babelstream(CMakePackage, CudaPackage, ROCmPackage):
             SIMPLE   - Recursively split its range until it cannot be further subdivided.\
             See https://spec.oneapi.com/versions/latest/elements/oneTBB/source/algorithms.html#partitioners for more details.",
     )
-
+    # OpenMP Intel Offload
+    variant("intel_target", values=str, default="none", description="Intel Offload target")
```

Comment:
Should this logic be unified with the `offload` variant above?

Reply:
Thank you for spotting this, will merge them. One quick question would be whether using fewer `variant` names and using them in multiple models is favoured. Since we have 11+ models, it is hard to reduce the number of `variant`s ?